olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -740,6 +765,9 @@ size_t NetworkCurl::HeaderFunction(char* ptr, size_t size, size_t nitems,
                                    RequestHandle* handle) {
   const size_t len = size * nitems;
 
+  OLP_SDK_LOG_TRACE(kLogTag, "Received header for id=" << handle->id
+                                                       << "; len=" << len);
```

Comment:
Does printing len provide important information?

Reply:
At least, it is better to print the length of the headers received cause it proves that the SDK have received not-null data - just printing the id doesn't make sense to me.

You suggest to print all headers?

Reply:
Ideally we will print headers and values. But this might be very verbose and will flood cout with countless logs. I dont think that headers will be empty so no need to print len. Better remove. We can add if desired.

Reply:
Ok, removed